src/commons/dropdown/DropdownCourses.tsx

```diff
@@ -15,6 +16,7 @@ type Props = {
 
 const DropdownCourses: React.FC<Props> = ({ isOpen, onClose, courses, courseId }) => {
   const navigate = useNavigate();
+  const enableExamMode = useTypedSelector(state => state.session.enableExamMode);
```

Comment:
We can use `useSession` for simplicity:

```suggestion
  const { enableExamMode } = useSession();
```